patches/isrg-cp-from-brs.patch

```diff
@@ -203,8 +203,8 @@ index 5823995..280f22d 100644
 +
 +Policy Management Authority<br/>
 +Internet Security Research Group<br/>
-+548 Market St., PMB 57274<br/>
-+San Francisco, CA 94104-5401<br/>
++P.O. Box 18666<br/>
++Minneapolis, MN 55418-0666</br>
```

Comment:
This changed from a self-closing `<br/>` to a normal-closing `</br>`, even though it has no opening tag. The same in the two other locations.

Honestly all of these should be changed to the standard `<br>` with no slash at all.